lighthouse-core/report/html/renderer/util.js

```diff
@@ -487,6 +487,14 @@ class Util {
   }
 }
 
+/**
+ * Some parts of the report renderer require data found on the LHR. Instead of wiring it
+ * through, we have this global. This property is optional so that existing rendering clients
+ * won't break.
```

Comment:
What existing rendering clients would this not break that won't be broken by the required `Util.i18n` right below this, also releasing in 6.0?

Reply:
I'm thinking anything that uses Category Renderer directly. psi? idk, prob others.

Reply:
> I'm thinking anything that uses Category Renderer directly. psi? idk, prob others.

Which can roll a new `performance-category-renderer` and then throw on the first reference to `Util.i18n.strings`. It's not clear why this clarification is needed when there isn't anyone upgrading with no work.